private/tools/java/com/github/bazelbuild/rules_jvm_external/resolver/remote/HttpDownloader.java

```diff
@@ -99,7 +99,7 @@ public static EventListener defaultEventListener() {
     boolean consoleAvailable = System.console() != null;
     if (System.getenv("RJE_VERBOSE") != null) {
       return new PlainConsoleListener();
-    } else if (termAvailable && consoleAvailable) {
+    } else if (termAvailable && consoleAvailable || System.getenv("FORCE_ANSI") != null) {
       return new AnsiConsoleListener();
```

Comment:
I wanted to test the AnsiConsoleListener in a test to validate the cleanup; unfortunately Bazel test runner calls `System.exit` making it difficult to test it. [[slack](https://bazelbuild.slack.com/archives/CDCE4DFAP/p1756831399977859)]